### PR TITLE
add federal agency fields and validation

### PR DESCRIPTION
**Pull request by cpapazian**
all set ...

### CODE REVIEW

**Comment by cpapazian** [comment]
@polastre or @yozlet want to sign off on this?

File: app/controllers/applications_controller.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class ApplicationsController < ApplicationController
+  # helper FormErrorsHelper
```
Comment (polastre) [inline app/controllers/applications_controller.rb]: remove comment.

**Comment by polastre** [comment]
I don't see any JS that controls showing the "I agree" checkbox after selecting "I'm a federal agency".  How does that work?

File: app/assets/javascripts/myusa.js [hunk 2]
```diff
@@ -146,4 +146,13 @@ $(document).ready(function () {
...
 
+  $('#application_federal_agency').click(function (e) {
+    var formGroup = $('div.form-group').has('input#application_terms_of_service_accepted');
+    if (e.target.checked) {
+      formGroup.show();
+    } else {
+      formGroup.hide();
+    }
+  });
+
```
Comment (cpapazian) [inline app/assets/javascripts/myusa.js]: @polastre ^^^ toggles the TOS checkbox based on whether the federal agency checkbox is set.

Comment (polastre) [inline app/assets/javascripts/myusa.js]: :+1:

**Comment by polastre** [comment]
:shipit: :+1: :clap:

**Pull request merged**